src/reference/modules/installation/pages/_partials/debian_repo.adoc

```diff
@@ -8,7 +8,7 @@ Add Rudder's package repository:
 #   stretch for Debian 9
 #   bionic  for Ubuntu 18.04 LTS
 
-echo "deb http://repository.rudder.io/apt/7.3/ $(lsb_release -cs) main" > /etc/apt/sources.list.d/rudder.list
+echo "deb [arch=amd64] http://repository.rudder.io/apt/7.3/ $(lsb_release -cs) main" > /etc/apt/sources.list.d/rudder.list
```

Comment:
this will not work on arm systems

Reply:
there is no arm system on public repository

Reply:
indeed